### ISSUE TITLE
Order of multiple hooks: mismatch of documentation and behavior

### ISSUE THREAD

**Issue by Mairu** [feathersjs/hooks#38]
### Steps to reproduce

Running the script from the documentation: https://github.com/feathersjs/hooks#middleware

### Expected behavior

The output of the script should be (documented):
```
hook1 before
hook2 before
hook3 before
Hello David
hook3 after
hook2 after
hook1 after
```

### Actual behavior

The output of the script is:
```hook3 before
hook2 before
hook1 before
Hello David!
hook1 after
hook2 after
hook3 after
```

### Thoughts

As hooks are decorators, it is clear that the last wrapped hook is run first.
So the question is, if the documentation should be adjusted, or if the order of the array should be reversed for wrapping. So that the first entry is wrapped as last one.

Brought up in Slack: https://feathersjs.slack.com/archives/C0HJE6A65/p1591009725097800?thread_ts=1590992656.097000&cid=C0HJE6A65

**Comment by daffl** [comment]
I believe this should be fixed via https://github.com/feathersjs/hooks/pull/37. There was a `middleware.reverse()` in there (https://github.com/feathersjs/hooks/commit/f597ad1146fe5675888fff3677605400cabb8cac#diff-dfef6fe5d351adc488c7440e64968cd7R53) that definitely did not belong there.

**Comment by daffl** [comment]
Published as v0.5.0
